site/docs/components/file-drop-zone/examples.mdx

```diff
@@ -0,0 +1,46 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="file-drop-zone" exampleName="Default">
+
+    ### FileDropZone
```

Comment:
This title should match the example name or the display name

Reply:
sorry, a bit confused... @mikearildbrown `File drop zone` or `Default`?